build-tools/src/main/resources/build-tools/pmd/pmd-ruleset.xml

```diff
@@ -88,6 +88,7 @@ limitations under the License.
     <rule ref="category/java/multithreading.xml">
         <exclude name="AvoidSynchronizedAtMethodLevel" />
         <exclude name="UseConcurrentHashMap" />
+        <exclude name="DoNotUseThreads" />
```

Comment:
Don't do this. There is a way to ignore specific methods only. We really shouldn't be creating threads especially in Jetty.

Reply:
Cool. I'll change it.